### PR TITLE
add favourites page

### PR DESCRIPTION
**Pull request by PaninDmytro**
Desktop: https://prnt.sc/vHszbm5HAHR2

Tablet: https://prnt.sc/UFajuNy8CZgO

Mobile: https://prnt.sc/oUP2fa4G4LHU

### CODE REVIEW

File: src/modules/FavouritesPage/FavouritesPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@import '../../styles//blocks/grid';
+@import '../../styles/colors';
+@import '../../styles/typography';
+@import '../../styles/mixins';
+@import '../../styles/blocks/container';
```
Comment (Clover247) [inline src/modules/FavouritesPage/FavouritesPage.module.scss:5]: Can you import all of that by @import './../../App.module.scss';?

File: src/modules/FavouritesPage/FavouritesPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.textFavourites {
+  color: $color__secondary;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.main_favourites {
+  font-family: 'Mont';
```
Comment (Clover247) [inline src/modules/FavouritesPage/FavouritesPage.module.scss:42]: Can you put it in .container for all classes?

**Review by Clover247 — APPROVED** [review]
Great job!

File: src/modules/FavouritesPage/FavouritesPage.module.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }
+}
+
+.itemsAmount {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 21px;
+  margin-top: 8px;
+  color: $color__secondary;
+}
```
Comment (BetterSol) [inline src/modules/FavouritesPage/FavouritesPage.module.scss:65]: maybe better to create `@mixin` for fonts? Or fix the order of attributes https://9elements.com/css-rule-order/

**Review by BetterSol — APPROVED** [review]
LGTM, suggested minor changes)

**Pull request merged**